src/modules/Accordion/AccordionTitle.js

```diff
@@ -43,17 +50,29 @@ export default class AccordionTitle extends Component {
     parent: 'Accordion',
   }
 
+  static create = createShorthandFactory(AccordionTitle, content => ({ content }))
```

Comment:
This method will need to be placed after the class is created, otherwise, AccordionTitle will be undefined in this context:


```jsx
class AccordionTitle extends Component {
  // ...
}

AccordionTitle.create = createShorthandFactory(AccordionTitle, content => ({ content }))
```

There is also a common test for this:

**AccordionTitle-test.js**
```js
common.implementsCreateMethod(AccordionTitle)
```